metadata.json

```diff
@@ -35,6 +35,10 @@
     {
       "name": "katello/certs",
       "version_requirement": ">= 6.0.0 < 7.0.0"
+    },
+    {
+      "name": "theforeman/pulpcore",
```

Comment:
This isn't released yet so it probably breaks things. I was actually thinking about adding it to `.fixtures.yml` first so you can at least run unit tests.

Reply:
Thanks - I've added `pulpcore`, `postgresql`, and `redis` to `.fixtures.yml`

Reply:
As discussed, we'll not do a release and keep it as a soft dependency for now. Let foreman-installer pull it in for us.